### DISCUSSION TITLE
Table key to link ADM1 and ADM2?

### DISCUSSION THREAD

**Discussion by danac5150** [wmgeolab/geoBoundaries#1910] [Q&A] [answered]
Hi, I just downloaded CGAZ4 (shpfile version) and was wondering if there is a field key that links the ADM1 and ADM2 datasets?  E.g. a field value that tells you a particular ADM2 unit belongs to an ADM1 unit.  Or can they only be related spatially?  Apologies if this is answered somewhere, I haven't been able to find metadata explaining the field values.  Thanks for your consideration and for making available and maintaining this wonderful dataset!

**Comment by DanRunfola** [comment, marked as answer]
Hi @danac5150 !  In our last version of CGAZ we provided a hierarchical join field, but it was just a spatial lookup done on our side.  We could recreate that, but it's a bit misleading as we (today - on the list) don't enforce hierarchical standardization in our data.  I.e., if our ADM3 is from 2021, our ADM2 might still be from 2016, leading to incongruent hierarchies.

I'd be open to thoughts on the value of introducing the spatial-lookup joins, though.  Just not sure if they're helpful or just misleading.

**Reply by danac5150** [reply]
Hi @DanRunfola, I see what you mean about the lookup tables being potentially misleading.  I think they could still be valuable for users like me to need to link the different ADM levels.  Maybe you could make them available and just provide a disclaimer?

Anyways, good to know, thanks for your response!

**Comment by DanRunfola** [comment]
Let me put this in as an issue for the next version.  There are a number of other issues I want to work on, so could wrap this in.

Date TBD :)
